src/lib.rs

```diff
@@ -132,12 +131,12 @@ impl Build {
 #[derive(Debug, Clone, StructOpt)]
 pub struct Watch {
```

Comment:
If all fields are public, I think you need to make your structs as non exhaustive so we don't have to break compatibility in future versions. https://doc.rust-lang.org/reference/attributes/type_system.html